.vscode/tasks.json

```diff
@@ -6,8 +6,8 @@
         {
             "label": "client-npm-compile",
             "dependsOn": [
-                "client-npm-compile:shell",
-                "client-npm-compile:package"
+                "client-npm-compile:package",
```

Comment:
Changed the order because if you changed APIs on the src side and then reacted on the test side the test project would continuously fail to build because it wouldn't see the new changes.